docs/ldml/tr35.md

```diff
@@ -1125,7 +1125,7 @@ In the transformed content 't' data file, the `name` attribute in a `<key>` elem
 ```xml
 <key extension="t" name="m0" description="Transform extension mechanism">
     <type name="ungegn" description="United Nations Group of Experts on Geographical Names" since="21"/>
-<key>
+</key>
```

Comment:
@srl295 - Do the tests in GitHub check for missing closing tags in the docs?

Reply:
This actually is body text and not markup. It's in an xml preformatted block. I suppose those could have some kind of linting but that might be challenging.

But generally, no, there isn't any markdown lint, nor html lint against the generated output right now.  There probably would be _errors_ If something was completely broken.

Catching this particular block might be challenging, but can you file a ticket for checking the docs generally if you'd like to do so?